docker/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+#set -e
+
+if [ ${PGID+x} ]; then
+  echo "Adding user netdata to group with id ${PGID}"
+  addgroup -g "${PGID}" -S hostgroup 2>/dev/null
+  sed -i "s/${PGID}:$/${PGID}:netdata/g" /etc/group
+fi
+
+/usr/sbin/netdata -u netdata -D -s /host -p "${NETDATA_PORT}" "$@"
```

Comment:
Note to self: This probably should be wrapped into `exec` as per https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#entrypoint

Reply:
Done :)